examples/C#/README.md

```diff
@@ -0,0 +1,50 @@
+# Writing an SSE plugin using C#
+
+General background information is found here:
+
+* [Writing an SSE Plugin](../../docs/writing_a_plugin.md)
+* [Protocol Description](../../docs/SSE_Protocol.md) (API reference)
+* Documentation and tutorials for [gRPC](http://www.grpc.io/docs/) and [protobuf](https://developers.google.com/protocol-buffers/docs/overview)
+
+## Content
+* [Starting the server](#starting-the-server)
+* [RPC methods](#rpc-methods)
+    * [GetCapabilities method](#getcapabilities-method)
+    * [ExecuteFunction method](#executefunction-method)
+* ["Intelligent" functionality](#intelligent-functionality)
+
+There is currently only one sample plugin for C#.
+It demonstrates how to implement aggregate and scalar functions, and how to declare them in the return value of GetCapabilities.
+
+## Configuring QlikSense to use the sample gRPC server
+By default, the C# sample plug-in runs on port 50054, so for a QlikSense Desktop installation, the following should be added to settings.ini:
+
+[Settings 7] 
+
+SSEPlugin=CSharp_Basic_example, localhost:50054;
+
+Note that the string CSharp_Basic_example is the identifier that will prefix all plug-in functions when they are called from within Qlik.
+Use a different identifier for your own plug-in, and remember that this exact string has to be used for the supplied qvf file to work with the extension.
+
+For single-machine development and testing it is OK to use insecure communication, but for production scenarios you should use certificates. See [generate_certs_guide](../../generate_certs_guide/README.md) for more information on how to create and configure certificates.
+
+## Starting the server
+
+The C# gRPC sample server can be built and started from within the Visual Studio environment. It is implemented as a console program, with the Main function setting up a server object that is bound to a connector class.
+RPC methods are implemented in the connector class BasicExampleConnnector.
+
```

Comment:
Consider adding a few lines about how to change settings and what can be changed (grpchost/port and certs).
Also, add some information about how the protobuf generated files are created and what it is based on.